source/constructs/lib/live-streaming.ts

```diff
@@ -168,9 +168,9 @@ export class LiveStreaming extends cdk.Stack {
     /**
      * Mapping for sending anonymous metrics to AWS Solution Builders API
      */
-    new cdk.CfnMapping(this, 'AnonymousData', { // NOSONAR
+    new cdk.CfnMapping(this, 'AnonymizedData', { // NOSONAR
```

Comment:
What rule are we suppressing with `// NOSONAR`?
Is this the rule that wants you to assign any new object to a variable?

I think in this case, it would be cleaner to assign it to a variable so you can use later. The object has helper methods that make the code more maintainable.

#### For example:

Instead of doing this to reference the map...

```ts
cdk.Fn.findInMap('AnonymizedData', 'SendAnonymizedData', 'Data')
```

It's a bit cleaner and more maintainable to do this...

```ts
const anonymizedData = new cdk.CfnMapping(this, 'AnonymizedData', {
  mapping: {
    SendAnonymizedData: {
      Data: 'Yes'
    }
  }
});

// Then reference the map like this
anonymizedData.findInMap('SendAnonymizedData', 'Data')
```